src/component/feature/FeatureToggleList/FeatureToggleListTable/DateCell/DateCell.tsx

```diff
@@ -0,0 +1,32 @@
+import { VFC } from 'react';
+import { useLocationSettings } from 'hooks/useLocationSettings';
+import { formatDateYMD, formatDateYMDHMS } from 'utils/formatDate';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { Tooltip } from '@mui/material';
+
+interface IDateCellProps {
+    value?: Date | null;
+}
+
+export const DateCell: VFC<IDateCellProps> = ({ value }) => {
```

Comment:
Should this be somewhere in `components/Table` instead? Seems generic.

Reply:
Good point, but since we have just one table component now it will be easier to refactor when component is reused, not ahead of time.